requirements.txt

```diff
@@ -0,0 +1 @@
+asyncio-mqtt==0.8.0
```

Comment:
This requires Home Assistant to use the Selector Loop on Windows, which has limitations that we don't want.

Reply:
My reasoning was that since we'll only use the built in client with Supervisor, we can assume a Linux host.

Reply:
You are correct.